src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -193,7 +193,7 @@ namespace EvaporativeCoolers {
 
     // Functions
 
-    void SimEvapCooler(std::string const &CompName, int &CompIndex)
+    void SimEvapCooler(std::string const &CompName, int &CompIndex, Optional<Real64 const> PartLoadRatio)
```

Comment:
@Nigusse We're trying to avoid using optional arguments (for performance). A default value for the argument is better (if the argument is left off then it's set to the default value). See [discussion here](https://github.com/NREL/EnergyPlus/pull/7230#issuecomment-517287379). Then you don't need `if (present . . .)`. You can just use and pass `PartLoadRatio` directly.

Reply:
@mjwitte Okay, will modify the code accordingly. Thanks.

Reply:
done.